src/orchard/meta.clj

```diff
@@ -101,6 +104,13 @@
 ;; They just map to a special symbol.
 (def special-sub-symbs '{& fn*, catch try, finally try})
 
+(defn repl-special-meta
```

Comment:
This function has been added in order to dynamically require either `clojure.repl` or `cljs.repl` depending on what's available on the classpath - `require-and-var` should be able to avoid resolving the var every time and do it only once. I will make sure it is like that. Any comment/feedback on how to better require dynamically vars is greatly appreaciate here.

Reply:
@bbatsov @cichli I am intersted in your opinion on this, in particular how to make it faster